src/zimscraperlib/zim/indexing_item.py

```diff
@@ -0,0 +1,199 @@
+""" Special item with customized index data and helper classes """
+
+from __future__ import annotations
+
+import io
+import pathlib
+from typing import Any
+
+import libzim.writer  # pyright: ignore
+
+try:
+    import pymupdf
+except ImportError:  # pragma: no cover
+    import fitz as pymupdf
```

Comment:
`pymupdf` has been only very recently renamed from `fitz` to `pymupdf`. This is a fallback should someone prefer to use an "older" `pymupdf` version

Reply:
Our package clearly marks mypudf as a dependency so I dont see this helping much

Reply:
I think you don't get it, the Pypi package is named pymupdf since long ago, but the module was named fitz up till 1.24.3 (2024-05-09), see https://pymupdf.readthedocs.io/en/latest/changes.html ; I prefer to use pymupdf by default even if for now fitz is still supported, because we are never sure when this will end (and I don't want to be bothered by that in a future pymupdf update).

Reply:
Where's my comment?

Reply:
This is odd, all your comments appeared just right now on my side ... they are even all stacked in a single email notif from 5 mins ago. Something weird happened somewhere ...